resources/ports.yml

```
@@ -1443,6 +1443,12 @@ ports:
     color: mauve
     icon: obsidian
     current-maintainers: [*mbeckrich]
+  ohmyrepl:
+    name: OhMyREPL
+    categories: [cli, development, terminal]
```

The definition of terminal in our categories is an actual terminal so I'll remove it from this list, the other two seem okay though!

```suggestion
    categories: [cli, development]
```

---

Good to know, thanks for catching it!